relayer/msgs.go

```diff
@@ -320,3 +323,195 @@ func (c *Chain) MsgTransfer(dst *PathEnd, amount sdk.Coin, dstAddr string,
 		timeoutTimestamp,
 	)
 }
+
+// MsgRelayTimeout
+func (c *Chain) MsgRelayTimeout(counterparty *Chain, packet *relayMsgTimeout) (msgs []sdk.Msg, err error) {
+	var recvRes *chantypes.QueryPacketReceiptResponse
+	// TODO: try commenting out retries to reduce complexity
+	// retry getting commit response until it succeeds
```

Comment:
We can remove these comments. It is probably easiest to leave the retries for now